client/src/components/MapView.vue

```diff
@@ -39,6 +40,11 @@ export default defineComponent({
       popup: null as mapboxgl.Popup | null,
     };
   },
+  computed: {
+    routerLayer () {
```

Comment:
can you leave a comment as to what routerLayer represents

Reply:
Done!